src/mixins/name-request-mixin.ts

```diff
@@ -47,7 +47,7 @@ export default class NameRequestMixin extends Vue {
 
     // ensure NR is consumable
     const state = this.getNrState(nrResponse)
-    if (state !== NameRequestStates.APPROVED) {
+    if (state !== NameRequestStates.APPROVED && state !== NameRequestStates.CONDITIONAL) {
```

Comment:
Conditional is valid as long as consent is cleared (null), received (R), or waived (N). If not then state, here, is NEED_CONSENT, and this condition is true and an error is displayed. See line 91 below for logic.